rbc/irtools.py

```diff
@@ -54,9 +54,9 @@ def get_function_dependencies(module, funcname, _deps=None):
     return _deps
 
 
-class JITRemoteCPUCodegen(nb.targets.codegen.JITCPUCodegen):
+class JITRemoteCPUCodegen(nb.core.codegen.JITCPUCodegen):
```

Comment:
I think in the short-run we need to support numba 0.48 as well.
So, I suggest the following approach:
```
import numba as nb
if nb.__version__ >= (0, 49):
    from nb.core import codegen
else:
    from nb.targets import codegen
```
(or similar) and in the code use `codegen` instead of `nb.targets.codegen`

Reply:
Actually, the above will not work. Use:
```
try:
    from nb.core import codegen
except ImportError:
    from nb.targets import codegen
```